src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -701,6 +701,19 @@ private void WriteHeader(string name, string value)
             _headerBuffer.Commit(bytesWritten);
         }
 
+        private void WriteHeader(string name, string[] values, string separator)
+        {
+            // TODO: ISSUE 31307: Use static table for known headers
```

Comment:
Sorry @wfurt, my change that implemented this will end up conflicting with a bunch of your changes, but hopefully it'll be fairly straightforward for you to rebase and adapt.

Reply:
updated @stephentoub.  It seems like the biggest blocking issue is debate around StreamReader and underlying Loopback server.